src/main/groovy/com/okta/test/mock/Config.groovy

```diff
@@ -24,7 +24,10 @@ class TestScenario {
     List<String> disabledTests = new ArrayList<>()
     Boolean enabled
     String command
-    String loginRedirectPath = ""
+    /**
+     * A protected path that will force a redirect or an access error.
+     */
+    String protectedPath = "/"
```

Comment:
We should have a common pattern for protected paths. In our other libs we make this extremely clear by using `/protected`.

I'd suggest changing this to `/protected` instead of a root domain path.